import.csv

```diff
@@ -1,2 +1,5 @@
 organisation,documentation-url,endpoint-url,start-date,pipelines,plugin
-local-authority-eng:DUD,https://www.dudley.gov.uk/residents/planning/brownfield-land-register/,https://www.dudley.gov.uk/media/xxhj0cym/brownfield-land.csv,2023-10-16,brownfield-land,
\ No newline at end of file
+local-authority-eng:SWD,https://www.swindon.gov.uk/info/20113/local_plan_and_planning_policy/647/monitoring_and_evidence_base/3,https://www.swindon.gov.uk/download/downloads/id/6840/brownfield_land_register_appendix_1_-_december_2023.csv,2023-12-30,brownfield-land,
+local-authority-eng:BOS,https://www.bolsover.gov.uk/index.php?option=com_content&view=article&id=96,https://www.bolsover.gov.uk/component/edocman/14028-bdc-brownfield-land-register-2023-csv-format/download?Itemid=0,2024-01-02,brownfield-land,
+local-authority-eng:THE,https://www.threerivers.gov.uk/services/planning/planning-policy/registers#Brownfield%20land%20register,https://cdn.threerivers.gov.uk/files/2024/01/097a6320-abda-11ee-b20b-f5e0dc691cc7-brownfield-land.csv_nZzKHDbLsd3dGs0OIl-Urw_harmonised.xlsx,2024-01-05,brownfield-land,
+local-authority-eng:MOL,https://www.molevalley.gov.uk/planning-building/brownfield-land-register/,https://molevalleydc.sharepoint.com/:x:/s/MVDCDocumentLibrary/Ef_QIKlMYEdIuvN6MGVtsmcBo9pNhMKVIufZR-mzYx0FXA?download=1,2024-01-05,brownfield-land,
```

Comment:
I think having this file on github is confusing as it is a temporary file used for adding endpoints. I know you didn't add it but would you be able to delete it?